mllib/src/main/scala/org/apache/spark/ml/r/LDAWrapper.scala

```diff
@@ -172,6 +187,8 @@ private[r] object LDAWrapper extends MLReadable[LDAWrapper] {
       model,
       ldaModel.logLikelihood(preprocessedData),
       ldaModel.logPerplexity(preprocessedData),
+      trainingLogLikelihood,
+      logPrior,
```

Comment:
since model is referenced and persisted, is there a need to handle trainingLogLikelihood and logPrior separately like this, and writing to metadata, instead of just getting from the model when fetching for the summary?